### DISCUSSION TITLE
Pairing Does Not Provide Script

### DISCUSSION THREAD

**Discussion by fgams352** [openrport/openrport#19] [Q&A]
Thank You all for continuing this great project!

I updated the pairing site per the instructions provided. 

    sudo systemctl stop rportd.service
    Go to line 43-47 in /etc/rport/rportd.conf, change
    pairing_url = "https://pairing.openrport.io"
    Restart the rport server
    sudo systemctl start rportd.service

When I try to add a new client, I do not get the script to pair it. It looks like this:

![rport](https://github.com/openrport/openrport/assets/87881646/42d54ecc-0d5b-48a9-98ad-548df466ec24)

---------------------
It is supposed to look like this:

![rport2](https://github.com/openrport/openrport/assets/87881646/2e562f01-ada3-412d-95dd-159fb688d8a8)

How can I fix this?
Thanks!

**Comment by alhdo** [comment]
Hello to help you i need some more information. Which version of rport are you using ? And which version of the front-end.

**Comment by fgams352** [comment]
Hi Alhdo,

I've looked everywhere and cannot find the version anywhere. It is the last release from RealVNC before they pulled the project.

0.95?

**Comment by alhdo** [comment]
Did you restart the rportd service after ?

**Comment by Kwitchlang** [comment]
I think this may have to do with site caching - I had the same issue where I ended up reinstalling RPort multiple times on new LXC containers - and found a few things stayed the same, such as it is keeping me logged when the page was refreshed (running reverse proxy) even though it was a new server. as well as trying to run the New Client And the script wasn't showing up.

I remoted onto a server that wasn't having these issues - and it was able to generate the script, So I knew it wasn't anything to do with the server files itself.  

So maybe recommend clearing browser cache or when you set it up do it in incognito
